include/deal.II/dofs/dof_handler.h

```diff
@@ -1700,8 +1707,7 @@ class DoFHandler : public Subscriptor
   setup_policy();
 
   /**
-   * Setup connections to refinement signals of the underlying triangulation.
-   * Necessary for the hp-mode.
+   * Setup connections to signals of the underlying triangulation.
```

Comment:
We do this wrongly in many places, but the correct use of the word is "Set up connections...". For sure for verbs, it should be spelled apart. For nouns ("the setup we have here") might be spelled together, though "the set up of these connections" would probably still be apart.

Really not important here, but I've been finding these places all over the library.